src/Tribe/Integrations/Freemius.php

```diff
@@ -178,4 +180,34 @@ public function filter_connect_message_on_update(
 	public function get() {
 		return $this->instance;
 	}
+
+	/**
+	 * Method to remove the sticky message when the plugin is active for freemius
+	 *
+	 * @since  TBD
+	 *
+	 * @return void
+	 */
+	public function maybe_remove_activation_complete_notice() {
```

Comment:
Should we attach this to `admin_init@P1`? 
Make sure the methods and classes below exist before calling, to avoid fatals.